### ISSUE TITLE
[gltron, frozen-bubble]: worked well before (gfx and sound), now window popping up shows only black screen inside

### ISSUE THREAD

**Issue by star-buck** [blue-systems/pm-img#214]

**Comment by bhush9** [comment]
At some point it started showing graphics and sound... but then window is not fullscreen, and if I start it with fullscreen option, it freezes kwin..

/usr/games/frozen-bubble --fullscreen

**Comment by bhush9** [comment]
Also input works in tiny window it shows.

**Comment by star-buck** [comment]
so it works if we just dont start it fullscreen?

**Comment by bhush9** [comment]
Yes, though from launcher it doesn't start, you can launch it from konsole with /usr/games/frozen-bubble

**Comment by bhush9** [comment]
also, input is flaky... like hitting button twice makes it actually start thing.. probably related to tiny window..
